build/.vsts-ci.yml

```diff
@@ -253,6 +253,10 @@ jobs:
       name: 'sdk-c--ubuntu-18'
   displayName: 'linux_mbed'
   steps:
+  - script: |
```

Comment:
go up to the `_PREVIEW_VSTS_DOCKER_IMAGE` variable and make it `aziotbld/linux-c-ubuntu-mbed`